.claude/CELESTRA-WEBSITE-SETUP.md

```
@@ -0,0 +1,1258 @@
+# Add Tailwind CSS (optional but recommended for styling)
+npx astro add tailwind
+
+# Start dev server
+npm run dev
+```
+
+---
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)